src/functions.php

```diff
@@ -4,10 +4,13 @@
 /**
  * Debug function used to describe the provided value type and class.
  *
+ * @param mixed $input Any type of variable to describe the type of. This
+ *                     parameter misses a typehint because of that.
+ *
  * @return string Returns a string containing the type of the variable and
  *                if a class is provided, the class name.
  */
-function describe_type($input): string
+function describe_type(/* mixed */ $input): string
```

Comment:
what is this offering here?

Reply:
fixed in 129a4d86acec990ba533ac6906fd3eb934d95b15